core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -1870,31 +1975,45 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     assertEquals(2, numRecords3, "Expected consumer3 to consume from offset 1")
   }
 
-  @Test
-  def testConsumingWithEmptyGroupId(): Unit = {
+  @ParameterizedTest
+  @ValueSource(strings = Array("consumer")) // Null group ID not supported with consumer group protocol
+  def testNullGroupIdNotSupported(groupProtocol: String): Unit = {
```

Comment:
This name suggests that a group ID of `null` is not supported in general with manual assignment. However, `null` is supported as group ID with manual assignment. Just commits are not supported with group ID = `null`. Could you please use a more descriptive name for this test like `testNullGroupIdNotSupportedIfCommitting`?